src/utils/event-sourcing/src/event_id.rs

```diff
@@ -10,19 +10,19 @@
 /// A monotonically increasing identifier
 /// assigned by event stores to events
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct EventID(u64);
+pub struct EventID(i64);
```

Comment:
T_T

Reply:
Databases do not support unsigned types. If we keep `u64`, a lot gets complicated in queries.